VMBackup/main/fsfreezer.py

```diff
@@ -183,7 +190,7 @@ def thaw_safe(self):
                 else:
                     break
             self.logger.enforce_local_flag(True)
-            self.logger.log("Binary output after process end: ", True)
+            self.logger.log("============== Binary output traces start ================= ", True)
```

Comment:
this logging of binary output is being done when freeze is failed . Can you refactor it into separate method and add these tags there.

Reply:
refactored into a method.